pulpcore/cli/common/__init__.py

```diff
@@ -144,11 +144,18 @@ def _debug_callback(level: int, x: str) -> None:
         debug_callback=_debug_callback,
     )
     ctx.obj = PulpContext(api_kwargs=api_kwargs, format=format, background_tasks=background)
+    if ctx.invoked_subcommand == "shell":
+        make_click_shell(ctx, prompt="pulp-cli>", intro="Starting Pulp3 CLI...").cmdloop()
```

Comment:
Can't this command be executed instead of the `pass` in line 156?
Maybe with `ctx.parent` to run in the correct context...

Reply:
You are correct!